outputs.tf

```diff
@@ -29,6 +29,11 @@ output "reader_endpoint" {
   description = "A read-only endpoint of the DocumentDB cluster, automatically load-balanced across replicas"
 }
 
+output "cluster_members" {
+  value = aws_docdb_cluster.default[*].cluster_members
```

Comment:
is it possible that more than one `aws_docdb_cluster` is created by this module? If it isn't, then you should instead use the `one()` syntax used in cloudposse's modules or `flatten()` on this

Reply:
I've added the flatten()